doc/troubleshooting.md

```diff
@@ -609,6 +660,47 @@ Refer to [FAQ: Can names of joints be changed?](faq.md#can-names-of-joints-be-ch
 
 After correcting the configuration, the [changes will need to be propagated to the Yaskawa controller](../README.md#updating-the-configuration).
 
+### Alarm: 8011[56 - 58]
+
+Please refer to [Alarm: 8011[23 - 54]](#alarm-801123---54).
+
+### Alarm: 8011[59]
+
+*Example:*
+
+```text
+ALARM 8011
+ Host on NIC check x
+[59]
+```
+
+*Solution:*
+
+The `agent_ip_address` key in the `motoros2_config.yaml` configuration file is an address that is not reachable by the robot controller.
+
+Options:
+
+1. Modify the `agent_ip_address` key and specify an IP address that is on the robot's subnet.
+   Now follow the instructions [to propagate the changes to the Yaskawa controller](../README.md#updating-the-configuration).
+2. Modify the robot controller's IP so it is on the Agent's subnet.
+3. Modify the robot controller's network settings to add a gateway which can reach the Agent's IP address.
+
+Refer to the relevant Yaskawa Motoman documentation for information on how to change the controller's network configuration.
+
+### Alarm: 8011[60 - 62]
+
+*Example:*
+
+```text
+ALARM 8011
+ Must enable ETHERNET function
+[60 - 61]
```

Comment:
is it `[60 - 62]` as in the section header, or `[60 - 61]` as stated here.

Reply:
62